lib/ReactGridLayoutPropTypes.js

```diff
@@ -59,6 +59,7 @@ export type Props = {|
   width: number,
   autoSize: boolean,
   cols: number,
+  dragDelayDuration: number,
```

Comment:
`dragTouchDelayDuration` may be more clear

Reply:
Fine, I'll change the prop name.